stories/dooboo-ui/Select.stories.tsx

```diff
@@ -0,0 +1,278 @@
+          isDarkMode={false}
+          bordered={true}
+          activeOpacity={0.5}
+          placeholder={'New fancy select'}
+          value={selectedValue}
+          onSelect={(value): void => onSelectValue(value)}
+          onOpen={(isOpen): void => toggleSelect(isOpen)}>
+          <Item value={'Item-1'}>{'Item-1'}</Item>
+          <Item value={'Item-2'}>{'Item-2'}</Item>
+          <Item value={'Item-3'}>{'Item-3'}</Item>
+        </Select>
+      </CustomSelectContainer>
+    </CustomContainer>
+  );
+};
+
+const ChangeProps = (): React.ReactElement => {
+  const [darkTheme, setDarkTheme] = React.useState<boolean>(true);
+  const [borderless, setBorder] = React.useState<boolean>(true);
+  const [showArrow, setShowArrow] = React.useState<boolean>(true);
+  const [disabled, setDisabled] = React.useState<boolean>(false);
+
+  // [Select] states
+  const [selectedValue, onSelectValue] = React.useState<string | number>();
+  const [isOpen, toggleSelect] = React.useState<boolean>(false);
+  const [isLoading, setLoading] = React.useState<boolean>(false);
+
+  const delaySelect = (selectedItem): void => {
+    setLoading(true);
+    setTimeout(() => {
+      setLoading(false);
+      onSelectValue(selectedItem);
+    }, 700);
+  };
+
+  return (
+    <CustomContainer>
+      <TogglerWrapper>
+        <View style={{ alignSelf: 'center' }}>
+          <CustomText>{'Dark mode?'}</CustomText>
+          <ThemeSwitch
+            value={darkTheme}
+            onValueChange={(): void => setDarkTheme(!darkTheme)}
+          />
+          <CustomText>{'Bordered?'}</CustomText>
+          <ThemeSwitch
+            value={borderless}
+            onValueChange={(): void => setBorder(!borderless)}
+          />
+          <CustomText>{'Show default Suffix (Arrow)?'}</CustomText>
+          <ThemeSwitch
+            value={showArrow}
+            onValueChange={(): void => setShowArrow(!showArrow)}
+          />
+          <CustomText>{'disabled?'}</CustomText>
+          <ThemeSwitch
+            value={disabled}
+            onValueChange={(): void => setDisabled(!disabled)}
+          />
+        </View>
+      </TogglerWrapper>
+      <CustomSelectContainer>
+        <Select
+          opened={isOpen}
+          loading={isLoading}
+          disabled={disabled}
+          showArrow={showArrow}
+          isDarkMode={darkTheme}
+          bordered={borderless}
+          activeOpacity={0.5}
+          placeholder={'New fancy select'}
+          value={selectedValue}
+          onSelect={(value): void => delaySelect(value)}
+          onOpen={(isOpen): void => toggleSelect(isOpen)}>
+          <Item value={'Item-1'}>{'Item-1'}</Item>
+          <Item value={'Item-2'}>{'Item-2'}</Item>
+          <Item value={'Item-3'}>{'Item-3'}</Item>
+        </Select>
+      </CustomSelectContainer>
+    </CustomContainer>
+  );
+};
+
+const Customized = (): React.ReactElement => {
+  const [listHeight, onChangeListHeight] = React.useState<string>('');
```

Comment:
`onChangeListHeight` => `setListHeight`, since this is just a set action from React's `useState`.

Reply:
Also, why is this named `listHeight`? This is a value from the `TextInput`. It seems a little bit confusing.

Reply:
You're right, even though this doesn't affect the main source, it shouldn't have a confusing name.

So I change this from:
`listHeight` => `listHeightValue`
`onChangeListHeight` => `setListHeightValue`